domain/src/main/java/com/aconno/sensorics/domain/format/FormatMatcher.kt

```diff
@@ -91,6 +91,6 @@ class FormatMatcher(
             .forEach {
                 readingTypes.addAll(it.getFormat().keys)
             }
-        return readingTypes.sorted()
+        return readingTypes.distinct().sorted()
```

Comment:
Have you tested this?

Reply:
Yes, variables are pulled by name thru string and it doesn't matter what deserializer it comes from as long as it is the specified device